src/easyscience/fitting/minimizers/minimizer_lmfit.py

```diff
@@ -164,9 +165,7 @@ def convert_to_par_object(parameter: Parameter) -> LMParameter:
         :rtype: LMParameter
         """
         ## TODO clean when full move to
```

Comment:
Same here

Reply:
This we need to keep. The if / else statement should be deleted when fully moving to the new variable